lib/rules/array-callback-return.js

```diff
@@ -131,6 +131,33 @@ function isCallbackOfArrayMethod(node) {
     return false;
 }
 
+/**
+ * Retrieves the array method name given a node inside of it.
+ * Method starts from a given node, and goes up in the parent chain until it finds the array method name.
+ * @param {ASTNode} node A node inside an array method.
+ * @returns {string} the method name or null if not available.
+ */
+function getArrayMethodName(node) {
```

Comment:
Maybe we can just rename and modify `isCallbackOfArrayMethod` to return the method's name or `null`, instead of having an additional function which (partially) duplicates the logic from `isCallbackOfArrayMethod`?